tests/acceptance/TemplatesPagesLoadCest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+class TemplatesPagesLoadCest {
+  function loadTemplatesPage(\AcceptanceTester $I) {
+    $I->wantTo('Confirm template page loads and tabs can be clicked through');
+    $I->login();
+    $I->activateWooCommerce();
+    //get to Template Selection page
+    $I->amOnMailpoetPage('Emails');
+    $I->click('[data-automation-id="new_email"]');
+    $I->click('[data-automation-id="create_standard"]');
+    $I->waitForText('Select a responsive template');
+    //Standard email templates tab
+    $I->waitForElement('[data-automation-id="select_template_8"]');
```

Comment:
The `index` in `select_template_{index}` is kind of random, it can change if we reordered templates in the source code. That's why I don't think it's a good idea to use it to decide if a template is `Standard` or `Post Notification`. I would use the template's name instead. What do you think?

Reply:
I think using index is actually less brittle here than using the template name. The test isn't necessarily confirming the right templates show under the right tab, only that at least (x) templates are displaying on each tab. This should work even if we replace all the current templates, as long as we don't end up with fewer than what I'm testing for here.

Reply:
You're right, that makes sense.